common/lib/xmodule/xmodule/js/src/video/02_html5_hls_video.js

```diff
@@ -42,7 +42,11 @@
                 if (config.browserIsSafari) {
                     this.videoEl.attr('src', config.videoSources[0]);
                 } else {
-                    this.hls = new HLS({autoStartLoad: false});
+                    if (config.state.auto_advance) {
+			this.hls = new HLS({autoStartLoad: true});
+	            } else {
+                        this.hls = new HLS({autoStartLoad: false});
+	            }
```

Comment:
nit: please fix the spacing on this change to use 4 spaces instead of tabs.

Reply:
Oops! Sorry about that. My vim editor had some weird feature enabled. Corrected it now.